www/public/schemas/2025.10/resolver.json

```diff
@@ -0,0 +1,155 @@
+{
+  "type": "object",
```

Comment:
Note: I tested this schema locally, and works as-expected in VS Code. Some other editors may have different levels of how much they support, for example Zed doesn’t respect `oneOf` but VS Code does.

I’m treating this as a “better than nothing” approach. If there are any bugs, we can fix them as they are reported.